warehouse/integrations/middleware/sqlquerywrapper/sql.go

```diff
@@ -217,3 +219,12 @@ func (tx *Tx) Rollback() error {
 	}
 	return err
 }
+
+func (tx *Tx) Commit() error {
+	startedAt := time.Now()
+	err := tx.Tx.Rollback()
```

Comment:
Rollback? :sweat_smile:

Reply:
I guess this could have been easily spotted if we had a spy in our test and included an expectation on `Commit` being called.

Reply:
Added a test for round trip?